src/app/header/header.component.scss

```diff
@@ -7,7 +7,7 @@
   line-height: none;
 }
 
-.poinrer{
+.pointer{
```

Comment:
is it used somewhere?

Reply:
yes,
<img width="669" alt="pointer" src="https://user-images.githubusercontent.com/71029756/125098930-2b60ea80-e0e0-11eb-97d7-c86ace5594d7.png">

 I saw in header.component.html